styleguide/_themes/derek/scss/globalElements/mainnav.scss

```diff
@@ -275,23 +198,14 @@
   margin-top: 50px;
 }
 
-@media only screen and (min-width: $screen-sm-min) {
-  .navbar-searchicon {
-    margin-left: 12px;
-  }
-
-  .navbar-activeArrow {
-    margin: 0 0 0 -5px;
-  }
-}
-
 //main nav on desktop
-@media only screen and (min-width: $screen-md-min) {
+@media only screen and (min-width: $screen-xs-max) {
```

Comment:
Should this be `$screen-xs-max`, or `$screen-sm-min` since it's a `min-width` situation?  I know we've had some weird edge-case situations, but I don't think this is one of them, but I will test these scenarios more thoroughly later .